src/build.d

```diff
@@ -1447,23 +1386,38 @@ class DependencyRef
     }
 }
 
+/** Initializes an object using a chain of method calls */
+struct MethodInitializer(T)
```

Comment:
Why is this a template now?

Reply:
Makes it more general, could be added to phobos.  I try to structure the code in a way that general code could be factored out into a library.

Reply:
Okay